Yosemite/YosemiteTests/Mocks/Networking/Remote/MockWordPressThemeRemote.swift

```diff
@@ -4,24 +4,46 @@ import Foundation
 /// Mock for `WordPressThemeRemote`.
 ///
 final class MockWordPressThemeRemote {
-    private var stubbedThemes: [WordPressTheme] = []
-    private var stubbedError: Error?
+    private var stubbedSuggestedThemes: [WordPressTheme] = []
+    private var stubbedSuggestedThemeError: Error?
```

Comment:
np: perhaps need to also be plural `stubbedSuggestedThemesError` ?

Reply:
I'll update this in #11338 to avoid triggering CI again for this PR 😅